app/views/placements/schools/partner_providers/remove.html.erb

```diff
@@ -0,0 +1,25 @@
+<% content_for :page_title, t(".page_title", provider_name: @partner_provider.name) %>
+<%= render "placements/schools/primary_navigation", school: @school, current_navigation: :partner_providers %>
+<%= content_for(:before_content) do %>
+  <%= govuk_back_link(href: placements_school_partner_provider_path(@school, @partner_provider)) %>
+<% end %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <label class="govuk-label govuk-label--l">
+        <span class="govuk-caption-l"><%= @partner_provider.name %></span>
+        <%= t(".are_you_sure") %>
+      </label>
+      <%#= render GovukComponent::WarningTextComponent.new(
```

Comment:
This one as well

Reply:
Those are left there on purpose, as a remind to add the warnings when doing the second part
https://github.com/DFE-Digital/itt-mentor-services/pull/452